core/src/com/fourinachamber/fortyfive/utils/Utils.kt

```diff
@@ -169,6 +169,8 @@ fun Collection<Timeline>.collectTimeline(): Timeline {
     return Timeline(actions)
 }
 
+fun IntRange.midPoint(): Int = first + ((last - first) * 0.5).toInt()
```

Comment:
only minor, but with bit operator more effitient, since multiplieing by a decimal takes longer (or at least divide by 2 and not 0.5)